src/main/java/dao/FileWriterImpl.java

```diff
@@ -0,0 +1,24 @@
+package dao;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class FileWriterImpl implements dao.FileWriter {
```

Comment:
The class `FileWriterImpl` should be in a package that reflects its purpose. Consider placing it in a package named `service.impl` to follow the project structure guidelines.